ci/build.sh

```diff
@@ -8,4 +8,4 @@ if [ ! -d "$HOME/openjfx/javafx-sdk-11" ]; then
   ls -lah $HOME/openjfx/javafx-sdk-11/lib
 fi
 
-./gradlew --stacktrace -Pversion=${APP_VERSION} -PjavafxPlatform=linux jar
+./gradlew --full-stacktrace -Pversion=${APP_VERSION} -PjavafxPlatform=linux jar
```

Comment:
Is this relevant?

Reply:
No sorry was for debugging and I forgeot itthx